src/routes/Routes.tsx

```diff
@@ -27,6 +28,7 @@ export type RoutePath =
   | "/tteokguk/create"
   | "/my-page"
   | "/my-page/activity"
+  | "/search/user"
```

Comment:
```suggestion
  | "/user/search"
```

`/nickname/create`, `/tteokguk/create` 와 같은 컨벤션이라면 도메인이 먼저 나오고 행위가 나오는게 맞을것 같습니다!

Reply:
아 오 앞에 나오는게 도메인이라는 걸 인지 못하고 있었네요ㅠ배워갑니다ㅎㅎ